subprojects/docs/src/docs/release/notes.md

```diff
@@ -30,6 +30,12 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 
 TBD - load from cache performance improvements and reduced memory consumption for Android builds
 
+## Improved name matching
```

Comment:
Start with setting the context. For example: When running Gradle builds from the command-line, you can abbreviate project and task names. For example, you can execute task `compileTest` by running `gradle cT`.

Reply:
@pioterj I've rewritten the release notes entry, please have another look.